pkg/cri/resource-manager/policy/builtin/memtier/resources.go

```diff
@@ -191,15 +195,25 @@ type score struct {
 var _ Score = &score{}
 
 // newSupply creates CPU supply for the given node, cpusets and existing grant.
-func newSupply(n Node, isolated, sharable cpuset.CPUSet, granted int, normMem, grantedMem uint64) Supply {
+func newSupply(n Node, isolated, sharable cpuset.CPUSet, granted int, mem, grantedMem memoryMap) Supply {
 	return &supply{
 		node:                 n,
 		isolated:             isolated.Clone(),
 		sharable:             sharable.Clone(),
 		granted:              granted,
-		normMem:              normMem,
+		mem:                  mem,
 		grantedMem:           grantedMem,
-		extraMemReservations: make(map[Grant]map[memoryType]uint64),
+		extraMemReservations: make(map[Grant]memoryMap),
```

Comment:
Is this getting restored somehow on startup after we have built the pools based on topology and reloaded the grants from the cache ? I could not spot it in the code...

Reply:
You're right -- the supply structs are not restored from the cache. I'll recalculate the extra reservations after restoring the allocations.